src/frontend/components/profileModal/ProfileModal.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { useState } from "react";
+import { EditProfile } from "../../components";
+import { useOutsideClickHandler } from "../../custom-hooks/OutsideClickHandler";
+import { setDisplayProfileModal } from "../../redux/slices/userSlice";
+import { AvatarWithName } from "../../components";
+import "./ProfileModal.css";
+
+const ProfileModal = ({ user }) => {
+  const { profileModalType, displayProfileModal } = useSelector(
+    (state) => state.user
+  );
+  const [users, setUsers] = useState([]);
+  const ref = useRef();
+  const { resetMenu } = useOutsideClickHandler(ref);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (resetMenu) {
+      dispatch(setDisplayProfileModal(false));
+    }
+  }, [resetMenu]);
+
+  useEffect(() => {
+    if (profileModalType === "following") {
+      setUsers(user?.following || []);
+    } else if (profileModalType === "followers") {
+      setUsers(user?.followers || []);
+    }
+  }, [user]);
+
+  return (
+    <React.Fragment>
```

Comment:
Instead of mentioning React fragment Can simply write it as `
``<> 
{JSX stuff}
</>
```